sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsConf.scala

```diff
@@ -864,6 +864,13 @@ object RapidsConf {
     .booleanConf
     .createWithDefault(true)
 
+  val ENABLE_ORC_FLOAT_TYPES_TO_STRING =
+    conf("spark.rapids.sql.format.orc.floatTypesToString.enable")
+    .doc("The float/double numbers in GPU have different precision with CPU. So when casting " +
```

Comment:
Comment should cover the where the casting happens, e.g. when reading ORC files, the required schema differs from the file schema.